src/themes/five/src/components/ServiceForm/index.js

```diff
@@ -134,6 +159,73 @@ const ServiceFormUI = (props) => {
     setCurrentProfessional(professional)
   }
 
+  const validateSelectedDate = (curdate, menu) => {
+    const day = moment(curdate).format('d')
+    setIsEnabled(menu?.schedule?.[day]?.enabled || false)
+  }
+
+  const getTimes = (curdate, menu) => {
```

Comment:
I saw this function in another component, please use the Utils file to manage this method, you don't need to implement the same code in many components